src/components/KonnectorEdit.jsx

```diff
@@ -27,12 +27,19 @@ export const KonnectorEdit = ({ t, account, connector, deleting, disableSuccessT
         messages={[t('account.message.error.global.description', {name: connector.name})]}
       /> }
 
-      <Tabs initialActiveTab='sync' className={styles['col-account-edit-tabs']}>
+      <Tabs
+        initialActiveTab={
+          error && error.message === ACCOUNT_ERRORS.LOGIN_FAILED
```

Comment:
As `error.message === ACCOUNT_ERRORS.LOGIN_FAILED` is used twice, could we use a variable ?

```js
const hasLoginError = error.message === ACCOUNT_ERRORS.LOGIN_FAILED
// ...
```